src/App.js

```diff
@@ -1,16 +1,42 @@
 import React from 'react';
 import './App.scss';
+import { Counter } from './components/Counter';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(prevState => (
+      { count: prevState.count + 1 }
+    ));
+  };
 
-  add100 = () => {};
+  add100 = () => {
+    this.setState(prevState => (
+      { count: prevState.count + 100 }
+    ));
+  };
 
-  increase = () => {};
+  increase = () => {
+    if (this.state.count % 5 === 0) {
+      this.add100();
+    } else {
+      this.addOne();
+    }
```

Comment:
```suggestion
    if (this.state.count % 5 === 0) {
      this.add100();
    } 
      this.addOne();
```
I've meant that there's no need in redundant 'else', because you should add 1 in any case, and add 100 if only % 5 === 0, but ok. It works the same without 'else'